### PR TITLE
[SPARK-6856] [R] Make RDD information more useful in SparkR

### PR DESCRIPTION
**Pull request by Jeffrharr**

### CODE REVIEW

**Comment by shivaram** [comment]
Jenkins, ok to test

**Comment by shivaram** [comment]
@His-name-is-Joof Could you also add a unit test for this ? Something similar to https://github.com/apache/spark/blob/master/R/pkg/inst/tests/test_sparkSQL.R#L637

**Comment by His-name-is-Joof** [comment]
How's that? Very new to contributing to large projects in general, so criticism welcome. Excellent bugtracker and starter bugs!

File: R/pkg/inst/tests/test_rdd.R [hunk 1]
```diff
@@ -643,3 +643,8 @@ test_that("collectAsMap() on a pairwise RDD", {
   vals <- collectAsMap(rdd)
   expect_equal(vals, list(`1` = "a", `2` = "b"))
 })
+
+test_that("show()", {
+  rdd <- parallelize(sc, list())
```
Comment (shivaram) [inline R/pkg/inst/tests/test_rdd.R]: Could you pass in `1:10` or `nums` (defined above) to parallelize instead of an empty list ?

**Comment by shivaram** [comment]
Thanks - this is looking good. I had minor comment inline. Also could you bring your PR up to date with the latest master branch ?

**Comment by shivaram** [comment]
LGTM. Thanks @His-name-is-Joof . Merging this

**Comment by shivaram** [comment]
Actually I just tried this out locally and have a minor comment: Could you add a new line character to the end of the `toString` ? Right now on the shell the output I get looks like

```
> a <- parallelize(sc, 1:20, 2L)
> a
ParallelCollectionRDD[0] at parallelize at RRDD.scala:450>
```

**Comment by His-name-is-Joof** [comment]
Thanks for your patience shivaram.

**Comment by shivaram** [comment]
Jenkins, retest this please

**Comment by shivaram** [comment]
Thanks for the update. I tried it out and change looks good. Waiting for Jenkins now

**Comment by shivaram** [comment]
@shaneknapp Jenkins doesn't seem to picking up the re-test request. Is this related to the outages we had yesterday ?

**Comment by shaneknapp** [comment]
I'm out of town for the weekend. I can check Sunday night.

**Comment by shaneknapp** [comment]
jenkins test this please

**Comment by shaneknapp** [comment]
jenkins test this please

**Comment by shaneknapp** [comment]
ok, fixed

**Comment by shivaram** [comment]
Jenkins, ok to test

**Comment by shivaram** [comment]
@His-name-is-Joof -- Could you let me know what your JIRA username is ? I would like to assign this issue to your

**Comment by His-name-is-Joof** [comment]
Joof

**Comment by shivaram** [comment]
@srowen Could you add Joof as a contributor on our JIRA ? The assignee auto-complete doesn't seem to pick this up right now

**Pull request closed without merging**